### PR TITLE
.gitignore: Add .pytest_cache to .gitignore

### LINKED ISSUE #2326: Add .pytest_cache to .gitignore
Add `.pytest_cache` to `.gitignore`

difficulty/newcomer

### PR DESCRIPTION
**Pull request by sanshay**
This adds `.pytest_cache` to `.gitignore`.

Closes #2326.

### CODE REVIEW

**Comment by sanshay** [comment]
What do I have to do? :/

**Comment by utkarsh2102** [comment]
Hi @sanshay,
The changes are as required.
There's a problem with your commit message. Please modify your commit according to the [commit guidelines](https://api.coala.io/en/latest/Developers/Writing_Good_Commits.html).
Find me on gitter or telegram by @utkarsh2102 for further queries.

**Comment by sanshay** [comment]
OKay, thanks.
Will fix asap.

**Comment by sanshay** [comment]
Made changes as asked.
Please review: @IpshitaC @jayvdb @utkarsh2102

**Comment by palash25** [comment]
> The changes looks good to me.

@utkarsh2102 please read the http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html before approving changes

> Made changes as asked.

No you haven't

@sanshay I think you are getting confused. You don't need to change the PR title you have to change the commit message instead

Use `git commit --amend`
Then change the message accordingly

**Comment by utkarsh2102** [comment]
@palash25, I was talking about the file changes. Take a look at "View Changes" and you'd know the changes I'm talking about.
However, I heartily apologise if you felt I'm talking about the commit message.

**Comment by palash25** [comment]
> However, I heartily apologise if you felt I'm talking about the commit message.

Oh I didn't realise that you were only talking about file changes. But since you approved it I thought the you approved the commit message too.

We only ack PRs when they are perfect and ready to merge for pointing out things separately we resort to reviews and comments.

So PRs should only be approved once they are ready to be merged and that is usually done by the maintainers. @utkarsh2102 :smile:

File: .gitignore [hunk 1]
```diff
@@ -43,3 +43,5 @@ Gemfile.lock
 venv
 bears/upload/
 package-lock.json
+.pytest_cache
+
```
Comment (sangamcse) [inline .gitignore]: Why did you add one extra blank line? 😃  We already have one blank line in .gitignore file.

Comment (sanshay) [reply .gitignore]: @sangamcse, it's fixed now :)

**Comment by sanshay** [comment]
@palash25 I'd resolve #2507 as well, as soon as this is merged.
@sangamcse the changes you requested as been made. Please review the same.

**Comment by utkarsh2102** [comment]
> So PRs should only be approved once they are ready to be merged and that is usually done by the maintainers. @utkarsh2102 :smile: 

@palash25, sorry for the same. I was requested to review and hence approved the file changes.
Would be taken care of everything the next time :smile:

**Comment by palash25** [comment]
@sanshay please squash your two commits into one

Read this http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by sangamcse** [comment]
@sanshay, you should at least take a look at each comments on your PR.
The changes I requested should be done using `git commit --amend` which @palash25 has mentioned in https://github.com/coala/coala-bears/pull/2506#issuecomment-392504380.
Any way, you have made 2 commits here. So you need to squash your commits into one. After that your PR will be ready to merge. 😉

**Comment by anctartica** [comment]
@sanshay can you please make the requested changes...shouldn't take much time

File: .gitignore (hunk 1 above)
Comment (jayvdb) [reply .gitignore]: No, you need to amend the original commit.

Only one commit per PR for a simple PR like this.

And that one commit must have a good commit messge.

Anyway ... unfortunately, this PR is no longer needed due to https://github.com/coala/coala-bears/pull/2531
